File: iOS/AppiraterPlugin/AppiraterPlugin.m
//
//  AppiraterPlugin.m
//  What To Brew
//
//  Created by James Stuckey Weber on 3/27/12.
//  Copyright (c) 2012 ChinStr.apps, All rights reserved.
//

#import "AppiraterPlugin.h"
#import "Appirater.h"

@implementation AppiraterPlugin
@synthesize callbackID;

- (void) sigEvent:(NSMutableArray*)arguments withDict:(NSMutableDictionary*)options
{
	[Appirater userDidSignificantEvent:YES];
}

@end

File: iOS/AppiraterPlugin/AppiraterPlugin.js
/**
 * Phonegap Appirater plugin
 * Copyright (c) James Stuckey Weber 2012
 *
 */
var AppiraterPlugin = {
    
     sigEvent: function() {
          return Cordova.exec("AppiraterPlugin.sigEvent");
     },
     foreground: function() {
          return Cordova.exec("AppiraterPlugin.foreground");
     }

};